examples/fourth/main.go

```diff
@@ -23,15 +27,26 @@ const (
 	amqpQueue        = "patron"
 	amqpExchangeName = "patron"
 	amqpExchangeType = oamqp.ExchangeFanout
-	awsRegion        = "eu-west-1"
-	awsID            = "test"
-	awsSecret        = "test"
-	awsToken         = "token"
-	awsEndpoint      = "http://localhost:4576"
-	awsQueue         = "patron"
+
+	// Shared AWS config
+	awsRegion = "eu-west-1"
+	awsID     = "test"
+	awsSecret = "test"
+	awsToken  = "token"
+
+	// SQS config
+	awsSQSEndpoint = "http://localhost:4576"
+	awsSQSQueue    = "patron"
+
+	// SNS config
+	awsSNSEndpoint = "http://localhost:4575"
+	awsSNSTopic    = "patron-topic"
 )
 
 var (
+	awsSQSSession *session.Session
```

Comment:
Maybe we should make the following chnages:
1. Sessions should not be globals, create functions
2. We might need only one session

Reply:
I will implement 1., but for 2. we need to different sessions as they both relate to a different endpoint